monero/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1.1
+id: meganero-monero
+category: Finance
+name: Monero Node
+icon: https://raw.githubusercontent.com/deverickapollo/umbrel-monero/dev/ui/src/assets/community-monero-icon.png
+version: "1.0.0"
+tagline: Run your monero node and independently store and validate every single Monero transaction with it.
+description: >-
+  Run your monero node and independently store and validate every single Monero transaction with it.
+  This is a full node that will download the entire Monero blockchain and store it on your Umbrel. 
+  This is the most secure way to run a Monero node, but it will take a long time to sync and will require a lot of storage space.
+developer: Monero
+website: https://www.getmonero.org/
+dependencies: []
+repo: https://github.com/deverickapollo/umbrel-monero
+support: https://github.com/deverickapollo/umbrel-monero/issues
+port: 9976
+submitter: Deverick
+submission: https://github.com/hashes4merkle/hundredsats
```

Comment:
redirects to https://github.com/deverickapollo/meganero - maybe it's better to have the actual link here?